src/l500/l500-depth.cpp

```diff
@@ -88,6 +88,9 @@ namespace librealsense
         depth_sensor.register_option(RS2_OPTION_APD_TEMPERATURE,
             std::make_shared <l500_temperature_options>(_hw_monitor.get(), RS2_OPTION_APD_TEMPERATURE));
 
+        depth_sensor.register_option(RS2_OPTION_HUM_TEMPERATURE,
```

Comment:
Shouldn't it be guarded by min_fw_version ?

Reply:
It is not a new FW command.